attendance/sal_info/api.php

```diff
@@ -239,7 +239,19 @@
 } else if ($action == 'update_user_profile_detail_by_id') {  //action to update employee profile detail.
     if (isset($PARAMS['user_id']) && $PARAMS['user_id'] != "") {
         $user_id = $PARAMS['user_id'];
-        $res = Salary::UpdateUserInfo($PARAMS);
+        $sal_details = Salary::getSalaryInfo($user_id);
+        $salary = array();
+        for( $i = 0; $i < count($sal_details); $i++ ){
+            if( isset($sal_details[$i]['total_salary']) && $sal_details[$i]['total_salary'] != "" ){
+                $salary[] = $sal_details[$i]['total_salary'];
+            }
+        }
+        if( sizeof($salary) > 1 ) {
+            $res = Salary::UpdateUserInfo($PARAMS);
```

Comment:
you had total  blocked this functionality of update user info.
It means user info can be only update if he has more than 1 salary added,